### ISSUE TITLE
Can Real-time Data of Sensor Gyroscope be Output in Main Window

### ISSUE THREAD

**Issue by TomLuosy** [introlab/rtabmap#447]
I want to make a conditional judgment based on the real-time data of the gyroscope for selective modeling.But I did not find the relevant data settings in the code.
 I look forward to your reply. Thank you.

**Comment by matlabbe** [comment]
No, there is no visualization of IMU data in the MainWindow. If you are using ROS, I suggest to visualize it in RVIZ, there are some plugins that can be downloaded to show IMU data.

For rtabmapviz or rtabmap standalone visualization, we would have to modify processOdometry() function, get the IMU data saved in the message, then add it somwhow to CloudViewer. The visualization model would have to be created in CloudViewer with VTK primitives, or by using [CloudViewer::addOrUpdateLine()](https://github.com/introlab/rtabmap/blob/9cb1e4bbc5763a992ebdea848be5ec610d7afdf2/guilib/src/CloudViewer.cpp#L1458) function with "arrow" enabled to show an arrow looking down for accelometer as an example. For gyroscope (angular velocity), not sure how you can represent it.
